samples/react/.storybook/preview.js

```diff
@@ -1,12 +1,19 @@
-import React, { cloneElement, Children } from 'react';
 import { addParameters, addDecorator } from '@storybook/react';
 import { withA11y } from '@storybook/addon-a11y';
 import { withSitecoreProps } from '@degjs/storybook-decorator-sitecoreprops';
 import { withTaffy } from '@degjs/storybook-addon-taffy';
 import '../src/css/styleguide/styleguide.css';
 import '../src/css/app.css';
 
+const cssReq = require.context('!!raw-loader!../src', true, /.\.css$/);
```

Comment:
Any benefit (performance or otherwise) to updating this path to just include the variables directory?

Reply:
That's a good question. I'll change it for now, and we can always go back to something more general once we've seen how people use it on a project